apps/backend/src/libs/plugins/file-plugin/libs/types/avatar-file.type.ts

```diff
@@ -0,0 +1,11 @@
+import { type ValueOf } from "~/libs/types/types.js";
+
+import { type ImageMimetype } from "../enums/enums.js";
+
+type AvatarFile = {
```

Comment:
```suggestion
type File = {
```
Can be other file as well in future